cmd/testrunner/sensei.go

```diff
@@ -0,0 +1,264 @@
+package main
```

Comment:
@johncantrell97 I might now be getting into a different class of hurdles:

```
node-start_1         | thread 'main' panicked at 'invalid bitcoind rpc config: Os { code: 111, kind: ConnectionRefused, message: "Connection refused" }', src/main.rs:200:14
node-start_1         | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
node-start_1         | manage your sensei node at http://127.0.0.1:5401/admin/nodes
node-start_1         | starting 02482907cbfcfa4ec558978bf336203f4af7c010383134b705fa60ab1c687461c9@127.0.0.1:9735
node-start_1         | thread 'sensei' panicked at 'assertion failed: `(left == right)`
node-start_1         |   left: `0c84467d65d0150756208a428f6a0cb281a7c1c9f594c8d7f556e8875a7f0fb7`,
node-start_1         |  right: `6080f1f6eea2bf23c045f549de8ec109fa8cb6e3407fcb6f32ddc884d925a27e`: Blocks must be connected in chain-order - the connected header must build on the last connected header', /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/lightning-0.0.108/src/ln/channelmanager.rs:5478:13
```

This should be reproducible by running `./run.sh sensei`

Reply:
Yes, the first one I think is when it starts a bit early and fails because bitcoind isn't running yet. Docker-compose will then restart, so yes, this could be considered separate runs. I believe it's unrelated to the second error.

The sensei db will be created new every time I run `run.sh`. But I did find out that the error isn't always happening. Appears to be timing-dependent. Perhaps because a whole bunch of blocks are generated in quick succession, something bad happens occasionnaly?